### PR TITLE
Tms/update setup instructions

### PR DESCRIPTION
**Pull request by tmikeschu**
## Overview
Per @ericfreese 's request, this is an updated list of instructions for setting up the environment for local development.

This also adds a link in the README to the new SETUP.

Thanks to @ericfreese for the guidance!

### CODE REVIEW

File: SETUP.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+5. Prepare postgres
+```
+initdb bfr-data/
+```
+
+  * If you have an existing database by chance,
+```
+ps aux | grep postgres
+kill -9 (number from below)
```
Comment (Fosome) [inline SETUP.md]: Careful using `kill -9`!  It's a last resort.  Better advocate `kill -15`. http://unix.stackexchange.com/questions/8916/when-should-i-not-kill-9-a-process

**Comment by Fosome** [comment]
@tmikeschu Can we just replace the instructions for setup in the readme?  Why a separate doc?

**Comment by tmikeschu** [comment]
@Fosome In case that original set up still works for some environments, I didn't want to erase it. Maybe just add both procedures to the readme?

File: SETUP.md (hunk 1 above)
Comment (virtuallybing) [inline SETUP.md]: @Fosome @tmikeschu  is it typical to have instructions for killing an existing database in a README? I'm under the impression that most READMEs assume that you're working from a clean slate in that regard.

**Comment by virtuallybing** [comment]
I set up my local environment using @tmikeschu's updated instructions without any problems. I agree with @Fosome that the outdated instructions should be replaced - maybe replace the following with instructions on running the seeds.rb file that was added a little while ago?

```
If you want to start with an empty schema, you can proceed as usual (rake db:setup, etc.), or you can
load a database dump from me or elsewhere. If you start with an empty schema, you'll want to start
by creating a Volunteer user with the admin bit set, and a first region e.g.:

$ rails console
r = Region.new
r.name = "Somewhere"
r.save

v = Volunteer.new
v.email = "jane.doe@gmail.com"
v.password = "changeme"
v.admin = true
v.regions << r
v.assigned = true
v.save
```

**Comment by cllns** [comment]
Thanks for doing this 😄 

I agree with @Fosome, if these instructions work for people (as @virtuallybing confirmed), let's just put them in the `README.md` in place of the current instructions. Personally, I'd probably miss `SETUP.md`.

File: SETUP.md (hunk 1 above)
Comment (rylanb) [inline SETUP.md]: To be honest, I have no idea why you would want to kill your running postgres server ever?

**Comment by rylanb** [comment]
I lied its -> https://github.com/boulder-food-rescue/food-rescue-robot/pull/91 now.

**Comment by rylanb** [comment]
Thank you for your help!

**Pull request closed without merging**